test/WormholeReceiver.t.sol

```diff
@@ -23,6 +23,8 @@ contract WormholeReceiverTestHarness is WormholeReceiver {
     public
     isRegisteredSender(sourceChain, senderAddress)
   {}
+
+  function replayProtectModifierFunc(bytes32 deliveryHash) public replayProtect(deliveryHash) {}
```

Comment:
May as well start using the naming convention that we'll change to now instead of adding work for ourselves later
```suggestion
  function exposed_replayProtect(bytes32 deliveryHash) public replayProtect(deliveryHash) {}
```

Reply:
https://github.com/ScopeLift/l2-flexible-voting/pull/39/commits/ca5bd0c521443aff2e0975e30b77fa818b97ac90